docs/nodes/troubleshooting.md

```diff
@@ -0,0 +1,119 @@
+# Troubleshooting Guide
+Setting up a [Core Channel Node](./core/index.md) or a [Compute Resource Node](./compute.md) can be a daunting task. This page is here to help you troubleshoot the most common issues.
+
+## Core Resource Nodes
+
+- Ensure to backup configuration files before making changes.
+- Monitor the node after each troubleshooting step to check for resolution.
+- Document each step taken for future reference or for support if needed.
+- If you are unable to resolve the issue, please reach out to the Aleph.im team on [Telegram](https://t.me/alephim) for support.
+
+### SQUASHFS Errors in Diagnostic VM
+#### Issue Summary:
+Users may encounter SQUASHFS errors indicating a failure to decompress data, suggesting possible corruption of the runtime diagnostic VM.
+
+#### Symptoms:
+Repeated SQUASHFS errors such as
+
+- `Failed to read block`
+- `Unable to read data cache entry`
+- `zlib decompression failed, data probably corrupt`
+
+related to a specific block.
+
+#### Probable Cause:
+The runtime of the new diagnostic VM appears to be improperly downloaded or corrupted.
+
+#### Recommended Solution:
+
+1. **Clear Cache**: Remove the cache of the problematic file using the diagnostic VM hash. This can be done by deleting the file located at /var/cache/aleph/runtime/RUNTIME_HASH.
```

Comment:
Use back-quotes everywhere relevant

```suggestion
1. **Clear Cache**: Remove the cache of the problematic file using the diagnostic VM hash. This can be done by deleting the file located at `/var/cache/aleph/runtime/RUNTIME_HASH`.
```

Reply:
Use a `$` to indicate that it's a variable ?

```suggestion
1. **Clear Cache**: Remove the cache of the problematic file using the diagnostic VM hash. This can be done by deleting the file located at /var/cache/aleph/runtime/$RUNTIME_HASH.
```